iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/pipe/task/PipeTaskCoordinator.java

```diff
@@ -66,7 +66,12 @@ public TSStatus createPipe(TCreatePipeReq req) {
   }
 
   public TSStatus startPipe(String pipeName) {
-    return configManager.getProcedureManager().startPipe(pipeName);
+    TSStatus status = configManager.getProcedureManager().startPipe(pipeName);
+    if (status == RpcUtils.SUCCESS_STATUS && pipeTaskInfo.clearExceptions(pipeName)) {
+      LOGGER.info("Pipe {} has started successfully, clear its exceptions.", pipeName);
+      configManager.getProcedureManager().pipeHandleMetaChange(true, true);
+    }
```

Comment:
Move it to StartPipeProcedureV2#executeFromOperateOnDataNodes to avoid concurrent modification caused by clearExceptions